azure-tests/pom.xml

```diff
@@ -5,13 +5,13 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>com.azure</groupId>
-    <artifactId>autorest-java</artifactId>
+    <groupId>com.azure.tools</groupId>
+    <artifactId>azure-autorest-parent</artifactId>
     <version>1.0.0-beta.1</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <groupId>com.azure</groupId>
-  <artifactId>autorest-azure-tests</artifactId>
+  <groupId>com.azure.tools</groupId>
+  <artifactId>azure-autorest-azure-tests</artifactId>
```

Comment:
It's a bit odd to have `azure` twice in the name. Are there any other names we can use here instead of azure for these tests?

Reply:
hmm I don't think it's weird but we can rename to `azure-autorest-arm-tests` since the autorest argument to generate these is `--azure-arm`

Reply:
Yeah, I like `arm-tests` better than `azure-tests` because it's clear that these are management related tests. However, we have started using `resourcemanager` as the new name for `arm` in track 2 libraries. So, another option is `azure-autorest-resourcemanager-tests`

Reply:
`arm` should be fine. Anyway if we do not release this package, we would have chance to change it later.